osf/management/commands/migrate_education_employment.py

```diff
@@ -0,0 +1,182 @@
+    if attribute_name == 'schools':
+        query = 'INSERT INTO osf_usereducation (_id, user_id, ongoing, degree, department, institution, end_date, start_date, created, modified, _order) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)'
+
+    if attribute_name == 'jobs':
+        query = 'INSERT INTO osf_useremployment (_id, user_id, ongoing, title, department, institution, end_date, start_date, created, modified, _order) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)'
+
+    with connection.cursor() as cursor:
+        for user in queryset:
+            entry = getattr(user, attribute_name)
+            for order, data in enumerate(entry):
+                start_date, end_date = get_dates(data)
+                ongoing = data.get('ongoing')
+                title_or_degree = data.get('title') or data.get('degree')
+                department = data.get('department')
+                institution = data.get('institution')
+                created = datetime.datetime.now()
+
+                if dry_run:
+                    logger.info('dry_run')
+                else:
+                    sql_params = [
+                        generate_object_id(),
+                        user.id,
+                        ongoing,
+                        title_or_degree,
+                        department,
+                        institution,
+                        end_date,
+                        start_date,
+                        created,
+                        created,
+                        order
+                    ]
+                    cursor.execute(query, sql_params)
+
+
+def reset_field_content(queryset, original_attribute, dry_run):
+    for entry in queryset:
+        user = entry.user
+        start_date = entry.start_date
+        end_date = entry.end_date
+        attributes = {
+            'institution': entry.institution,
+            'startYear': start_date.year if start_date else None,
+            'startMonth': start_date.month if start_date else None,
+            'endYear': end_date.year if end_date else None,
+            'endMonth': end_date.month if end_date else None,
+            'department': entry.department,
+            'ongoing': entry.ongoing,
+        }
+        if original_attribute == 'schools':
+            attributes['degree'] = entry.degree
+        elif original_attribute == 'jobs':
+            attributes['title'] = entry.title
+
+        user_entries = getattr(user, original_attribute, [])
+        user_entries.append(attributes)
+        if dry_run:
+            logger.info('User: {}, attributes: {}'.format(user, attributes))
+        else:
+            setattr(user, original_attribute, user_entries)
+            user.save()
+
+
+def put_jobs_and_schools_back(rows, dry_run, education, employment):
+
+    if education:
+        education_queryset = UserEducation.objects.raw('''SELECT * From osf_usereducation LIMIT %s''', [rows])
+        if not dry_run:
```

Comment:
These `dry_run` checks will prevent the downstream `dry_run` checks from being checked. They should be unnecessary.